packages/cli/lib/lib/webpack/render-html-plugin.js

```diff
@@ -50,8 +50,8 @@ module.exports = function(config) {
 			config,
 			url,
 			ssr(params) {
-				Object.assign(params, { url });
-				return config.prerender ? prerender({ cwd, dest, src }, params) : '';
+				Object.assign(props, params, { url });
+				return config.prerender ? prerender({ cwd, dest, src }, props) : '';
```

Comment:
https://github.com/developit/preact-cli/blob/next/packages/cli/lib/resources/template.html#L23

we could possibly just remove `params` and pass the whole `values` object from above as props here

Reply:
@ForsakenHarmony I did have that at first actually, but wasn’t sure if you’d want the entire configuration passed in. I don’t mind either way.

Reply:
it would potentially take that away yes, but I think the `prerender-urls` file is more flexible in general

Reply:
I updated it here, but wasn't sure if you wanted me to remove params altogether, or if you wanted me to merge those objects also.

https://github.com/developit/preact-cli/pull/740/files#diff-b26e9c1b1185752601692f2207489852R54

ie:

```Object.assign(values, params, { url })```